Darktrace/darktrace-threat-visualizer/ingest/parser.yml

```diff
@@ -34,10 +34,16 @@ stages:
         filter: "{{json_event.message.model.then.actions.alert}}"
       - set:
           host.id: "{{json_event.message.device.did}}"
-          host.hostname: "{{json_event.message.device.hostname}}"
+          host.name: "{{json_event.message.device.hostname}}"
+      - set:
+          host.name: "{{json_event.message.device.hostname.split()[-1]}}"
```

Comment:
Why not `user.name`? In that case, the hostname is the concatenation of the service and the username (or user email).